bin/sozo/tests/register_test.rs

```diff
@@ -21,13 +21,16 @@ async fn reregister_models() {
 
     let ws = ops::read_workspace(config.manifest_path(), &config)
         .unwrap_or_else(|op| panic!("Error building workspace: {op:?}"));
-    let dojo_metadata = dojo_metadata_from_workspace(&ws);
+    let dojo_metadata = dojo_metadata_from_workspace(&ws).expect(
```

Comment:
In tests we can unwrap right?

Reply:
i used expect since it gives a more helpful error message incase code panics there.